### DISCUSSION TITLE
I ditched expensive SolveigMM Splitter

### DISCUSSION THREAD

**Discussion by defencedog** [mifi/lossless-cut#1071] [Ideas]
This is a great application. Two things i certainly want & I can donate for the effort as well.

1] I must be able to hide the side panel showing segment info

2] Plz Plz make an Android counterpart. Will love to have this working on my Galaxy Tab using Samsung DEX

**Comment by mifi** [comment]
1. This is already possible. click the caret (`>`) next to Segments to export
2. unfortunately it is extremely hard to port it to android because it doesn't support electron. Creating a separate app is something I thought of but it's not yet implemented as it's a completely different project. see #818
